viewTask.html

```diff
@@ -222,8 +223,56 @@ <h4>Today</h4>
                 </div>
             </div>
         </div>
+
+        <!-- Create Task Modal Start-->
+        <div class="modal fade" id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
Change the id from "exampleModal" to something meaningful and unique. Id's in HTML should be unique. Also, change the "exampleModalLabel".